ibis/backends/flink/registry.py

```diff
@@ -282,6 +282,21 @@ def _floor_divide(translator: ExprTranslator, op: ops.Node) -> str:
     return f"FLOOR(({left}) / ({right}))"
 
 
+def _array(translator: ExprTranslator, op: ops.arrays.Array) -> str:
```

Comment:
Generally we use `ops.Thing` so that we aren't tied to the specific module name and also to type less :)

Reply:
Thanks, removed `.arrays` from the relevant operations.